pkgs/development/python-modules/flask-security-too/default.nix

```diff
@@ -82,6 +71,7 @@ buildPythonPackage rec {
     passlib
     importlib-resources
     wtforms
+    freezegun
```

Comment:
This is declared wrong, it is only used in tests

Reply:
Your're right! Thanks. I changed it